app/views/spree/admin/orders/_shipment.html.haml

```diff
@@ -85,3 +85,25 @@
         %td.actions
           - if can?(:update, shipment) && shipment.can_modify?
             = link_to '', '', :class => 'edit-tracking icon_link icon-edit no-text with-tip', :data => { :action => 'edit' }, :title => Spree.t('edit')
+      
+      %tr.edit-note.hidden.total
+        %td{ :colspan => "5" }
+          %label
+            = "#{Spree.t(:note)}:"
+          = text_area_tag :note, @order.note
+
+        %td.actions
+          = link_to '', '', :class => 'save-note icon_link icon-ok no-text with-tip', :data => { :action => 'save' }, :title => I18n.t('actions.save')
```

Comment:
A lot of old code is still using the old Ruby syntax but you can use the new one which is shorter and more consistent with our Ruby code:
```suggestion
          = link_to '', '', class: 'save-note icon_link icon-ok no-text with-tip', data: { action: 'save' }, title: I18n.t('actions.save')
```